javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,112 @@
+let personName = null;
+let toDo = [];
+
+function getReply(command) {
+
+    //----------My name command-------------
+    if (command.includes("Hello my name is")) {
+        personName = command.split(' ').splice(-1);
+        return "Nice to meet you  " + personName;
```

Comment:
Consider sending the same username twice, then  show some other message like `Hello again, Benjamin`.